sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -180,9 +180,13 @@ class Dataset[T] private[sql](
     // to happen right away to let these side effects take place eagerly.
     queryExecution.analyzed match {
       case c: Command =>
-        LocalRelation(c.output, queryExecution.executedPlan.executeCollect())
```

Comment:
Actually do we need to do this? most `Command`s are just local operations(talking with metastore).